app/src/main/res/layout/fragment_home.xml

```diff
@@ -0,0 +1,116 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.core.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:fillViewport="true">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <androidx.constraintlayout.widget.Guideline
+            android:id="@+id/guidelineHome1"
+            android:layout_width="0dp"
+            android:layout_height="@dimen/dp_1"
+            android:orientation="horizontal"
+            app:layout_constraintGuide_percent="0.35" />
+
+        <androidx.constraintlayout.widget.Guideline
+            android:id="@+id/guidelineHome2"
+            android:layout_width="0dp"
+            android:layout_height="@dimen/dp_1"
+            android:orientation="horizontal"
+            app:layout_constraintGuide_percent="0.42" />
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:layout_width="0dp"
+            android:layout_height="?actionBarSize"
+            android:background="@android:color/black"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+
+        <androidx.cardview.widget.CardView
+            android:id="@+id/cardTop"
+            android:layout_width="0dp"
+            android:layout_height="0dp"
+            app:cardCornerRadius="@dimen/dp_40"
+            app:layout_constraintBottom_toTopOf="@id/guidelineHome1"
+            app:layout_constraintLeft_toLeftOf="parent"
+            app:layout_constraintRight_toRightOf="parent"
+            app:layout_constraintTop_toTopOf="parent">
+
+            <androidx.constraintlayout.widget.ConstraintLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:background="@android:color/black">
+
+                <ImageView
+                    android:id="@+id/imageFavourite"
+                    android:layout_width="@dimen/dp_40"
+                    android:layout_height="@dimen/dp_40"
```

Comment:
có thể không fix cứng width/height được không?

Reply:
Ảnh này e để là ảnh vector , ko hiểu sao để wrap content thì nó to đùng ra

Reply:
> Ảnh này e để là ảnh vector , ko hiểu sao để wrap content thì nó to đùng ra

có cách để set kích thước trong constrain đó tìm hiểu thêm nhá :D

Reply:
Em thấy là cái icon này nó khá bé ấy ạ , nên dùng ko dùng đc percent , mà chắc ko ảnh hưởng tới tỉ lệ các màn hình